### ISSUE TITLE
No 1.29.1 support?

### ISSUE THREAD

**Issue by AdiQVR** [Goobwabber/MultiplayerCore#53]
Are the servers for 1.29.1 down because it just says that i need to update my Beat Saber version?

**Comment by qfoxb** [comment]
> The servers have been updated to 1.37+ with the update support for 1.28.0 and 1.29.1 was dropped as the changes to the underlying netcode were too substantial to reasonably maintain support.
> 
> You can find the latest version of MpCore for PC Beat Saber here: https://github.com/Goobwabber/MultiplayerCore/releases/latest And the latest version of BeatTogether for PC Beat Saber here: https://github.com/BeatTogether/BeatTogether/releases/latest

yeah just drop support for the version of the game that has the most mods

**Comment by Kevman95** [comment]
Hello, 

Most mods are now on newer versions. It is not best interest to continue 1.29.1 support. If this was done, the mod would not be timely updated, nor would we be able to provide new features that make the experience better.

It's in your best interest to upgrade. Plain and simple.
